Lux/src/gamma_lut.cpp

```diff
@@ -42,14 +44,14 @@ void printIEEE( flubber var )
     // Prints the IEEE 754 representation
     // of a float value (32 bits)
  
-    std::cout << var.raw.sign;
+    std :: cout << var.raw.sign;
```

Comment:
can consider generalizing this with ostream& parameter too. then can have "convenience" function wrapper that supplies cout so you don't have to type the extra parameter in the common case of printing to stdout similar to how
printf is a convenience wrapper are fprintf
(and both are special cases of sprintf in a sense)

Reply:
This is code I copied from https://www.geeksforgeeks.org/program-for-conversion-of-32-bits-single-precision-ieee-754-floating-point-representation/ - I'm not using it at the moment but keeping it around for debugging purposes in case I have trouble with gamma_LUT down the road.

I marked these routines as deprecated and I will remove them when I am satisfied with gamma_LUT's correctness.